### ISSUE TITLE
Frida crashes when enumerate_exports is called on module with no exports

### ISSUE THREAD

**Issue by H0r53** [frida/frida#521]
I am learning frida and I've been fighting with a crash for a while now. I finally figured out my issue. I wanted to view all exports within a binary (ideally all functions that the binary calls), so for each module in enumerate_modules I called enumerate_exports. If I'm working with a library that does not include any exports (such as a simple driver program), frida crashes within each environment (console, python, javascript). If there are no exports to enumerate I would assume the program would simply continue instead of faulting. I imagine this would be a fairly easy fix.

Here is some very simple python code to reproduce the issue based on the "hello" program in the frida tutorials (https://www.frida.re/docs/functions/):
```

from __future__ import print_function
import frida
import sys

# Attach to process
session = frida.attach("hello")

# Enumerate module exports
for mod in session.enumerate_modules():
        print("Module name: {}".format(mod.name))
        print([x for x in mod.enumerate_exports()])

```

**Comment by oleavr** [comment]
Thanks for reporting! Modules without any exports are indeed handled, but apparently not on all platforms :-) — which one is this on?

**Comment by H0r53** [comment]
I'm running frida in a VM with: Linux ubuntu 4.13.0-43-generic #48~16.04.1-Ubuntu SMP Thu May 17 12:56:46 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

The frida client I used came straight from pip

**Comment by oleavr** [comment]
Fixed in https://github.com/frida/frida-gum/commit/45da27f7f146ff77f6fba491cf45cb1d64887d3a – available in Frida 11.0.10 as soon as the build workers finish chewing on it. Thanks for your help in tracking this one down!
